spec/requests/providers/client_completed_means_spec.rb

```diff
@@ -59,7 +59,7 @@
         let(:submit_button) { { continue_button: "Continue" } }
 
         it "redirects to next page" do
-          expect(subject).to redirect_to(providers_legal_aid_application_no_income_summary_path)
+          expect(subject).to redirect_to(providers_legal_aid_application_means_identify_types_of_income_path)
```

Comment:
do these not need the application passed to them? as other similar tests
```suggestion
          expect(subject).to redirect_to(providers_legal_aid_application_means_identify_types_of_income_path(legal_aid_application))
```